.changeset/angry-pandas-jog.md

```diff
@@ -0,0 +1,15 @@
+---
+"@paprika/action-bar": major
+"@paprika/helpers": minor
+---
+
+- Add `initialValue` prop to the `<SearchInput />`
+- Fix the debouncing in `<SearchInput />`
+- Rename `debouncedValue` to `debounceDelay`
+- Add `useCallbackDebouncer`
```

Comment:
This is okay too, but even _better_ would be to run `yarn changeset` two times.  That will create two `md` files, and then each can be more focussed - so this "Add `useCallbackDebouncer`" note could be applied just to the changeset for `@paprika/helpers`.

Reply:
Done